extensions/positron-run-app/package.json

```diff
@@ -39,7 +39,8 @@
     "@types/mocha": "^10.0.8",
     "@types/node": "^22.5.4",
     "@types/sinon": "^17.0.3",
-    "@types/sinon-test": "^2.4.6"
+    "@types/sinon-test": "^2.4.6",
+    "typescript": "^4.5.5"
```

Comment:
this allows us to run `vsce package` in the `extensions/positron-run-app` directory to generate a VSIX, which can be loaded into Positron. This is helpful for faster extension dev iteration when running a pre-built reh-web version of Positron.